_posts/2020-04-21-a-token-of-our-affection-uswds-2.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "A token of our affection - A field guide USWDS 2"
+date: 2020-04-21
+authors:
+- christine-bath
+- laura-gerhardt
+tags:
+- web design system
+- lessons learned
+- cloud.gov
+excerpt: "We recently worked with the cloud.gov team to update their public site, cloud.gov, to United States Web Design System 2. The USWDS provided concepts we were able to use to translate designs into code a lot faster and deliver higher fidelity results once we understood how to use them."
+---
+
+We recently worked with the cloud.gov team to update their public site, [cloud.gov](https://cloud.gov/), to United States Web Design System 2 (USWDS 2). This was our first time digging into Version 2 of the system.
```

Comment:
I realize we don't have a link to USWDS. Can update this line to add one? Suggestion below:

`We recently worked with the cloud.gov team to update their public site, [cloud.gov](https://cloud.gov/), to [United States Web Design System 2 (USWDS 2)](https://designsystem.digital.gov/).`

Reply:
Added